add 3 events to calendar (#2664)
mainmatter/mainmatter.com: src/calendar/2025-06-03-karlsruher-entwicklertag.md

---
title: "Karlsruher Entwicklertag 2025"
image: "/assets/images/calendar/2025-06-03-karlsruher-entwicklertag/rust.png"
location: "Karlsruhe"
url: https://entwicklertag.de/programm/BackendEntwicklungmitRust
kind: conference
color: purple
hero:
  image: "/assets/images/calendar/2025-06-03-karlsruher-entwicklertag/hero.png"
  imageAlt: "View of the city of Frankfurt"
---

> Marco Otte-Witte, our Managing Director, will be speaking at Karlsruher Entwicklertag 2025. He’ll explore why Rust is a real option for backend development, discussing its benefits, challenges, and real-world examples. The talk will be presented in German. Join the session and learn how Rust brings stability, performance, and safety to backend systems.

Reviewer: ```suggestion
Mainmatter Founder Marco Otte-Witte will be speaking at Karlsruher Entwicklertag 2025. He’ll explore why Rust is a real option for backend development, discussing its benefits, challenges, and real-world examples. Learn how Rust brings stability, performance, and safety to backend systems.
```
